src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #include "ray/gcs/gcs_server/gcs_resource_manager.h"
+
```

Comment:
can you please remove this change/diff?

Reply:
hmm, `scripts/format.sh` insists on this change

Reply:
you might have the wrong version of the c++ linter installed?
-- I experienced this kind of inconsistency once while picking a PR into a release and had to uninstall and install the precise version -- 
clang format 7.0.0
https://releases.llvm.org/download.html